### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by kevchant** [Frank-Geisler/powerrti#2] [Ideas]
I think it would be good for others if this repo had a populated Wiki for it as well. What do others think?

**Comment by Frank-Geisler** [comment]
Hi @kevchant !

This is a brillant idea and thank you for already creating a wiki. So there is some documentation in [documentation](https://github.com/Frank-Geisler/powerrti/tree/main/documentation). Shall we move this to the wiki? If so: how? The documentation is mostly auto generated from the PowerShell comments via platyps. We have to include this somehow in the cmdlets as well, so if you do something like `Get-Help mycmdlet -Online` that you will be taken there. At the moment I have no idea how this works.
